stdlib/transactions/src/main/ballerina/src/transactions/commons.bal

```diff
@@ -326,8 +330,7 @@ public function registerParticipantWithRemoteInitiator(string transactionId, str
 
     // Register with the coordinator only if the participant has not already done so
     if (participatedTransactions.hasKey(participatedTxnId)) {
-        string msg = "Already registered with initiator for transaction:" + participatedTxnId;
-        log:printInfo(msg);
+        log:printDebug("Already registered with initiator for transaction:" + participatedTxnId);
```

Comment:
@anupama-pathirage @afkham what do you think about going from log at info level to debug level in transaction coordinator?

Reply:
ATM there are considerable amount of info logs per single transaction. Most of we can add as debug. Anyway we can add that as separate task . All the new logs I added as debug.